src/engine/source/kvdb2/include/kvdb2/kvdbExcept.hpp

```diff
@@ -0,0 +1,79 @@
+#ifndef _KVDB_EXCEPT_HPP
```

Comment:
I think it should be `_KVDB_EXCEPT_H`

Reply:
Will be fixed in later documentation step.